cmd/podman/containers/clone.go

```diff
@@ -68,6 +71,10 @@ func clone(cmd *cobra.Command, args []string) error {
 			ctrClone.RawImageName = rawImageName
 		}
 	}
+	if ctrClone.Force && !ctrClone.Destroy {
```

Comment:
```suggestion
	if ctrClone.Force {
	       if !ctrClone.Destroy && userSetItToFalse() {
	            return err
	       }
```
With `userSetItToFalse()` being replaced with the check if it was specified on the command line.

`--force --destroy=false` should return an error.

Reply:
It might be my lack of playing with Gingko lately, or too many reviews today, but doesn't `if !ctrClone.Destroy` return TRUE if the `--destroy` option is not presented or if it is set to false by the user?  `--destroy=false`?

Reply:
@TomSweeneyRedHat we decided that the user cannot specify `--force` without `--destroy` so either `--destroy=false --force` or `--force` should fail